kratos/geometries/geometry.h

```diff
@@ -634,6 +634,99 @@ class Geometry
         return mData.GetValue(rThisVariable);
     }
 
+    ///@}
+    ///@name Dynamic access to internals
+    ///@{
+
+    /* Set s a certain value to the geometry,
+     * according to a variable.
+     * Allows dynamic interfaces with each respective geometry.
+     */
+
+    /// Set with bool
+    virtual void Set(
+        const Variable<bool>& rVariable,
+        bool& Output) {}
+
+    /// Set with int
+    virtual void Set(
+        const Variable<int>& rVariable,
+        int& Output) {}
+
+    /// Set with double
+    virtual void Set(
+        const Variable<double>& rVariable,
+        double& Output) {}
```

Comment:
```suggestion
        const double Input {}
```
this waythe usage is clearer

Reply:
definitely. Thanks. I changed it